drivers/video/ov7670.c

```diff
@@ -547,10 +550,73 @@ static int ov7670_init(const struct device *dev)
 	return 0;
 }
 
+static int ov7670_stream_start(const struct device *dev)
+{
+	return 0;
+}
+
+static int ov7670_stream_stop(const struct device *dev)
+{
+	return 0;
+}
+
+static int ov7670_set_ctrl_vflip(const struct device *dev, bool enable)
+{
+	int ret;
+	uint8_t reg = OV7670_MVFP;
+	const struct ov7670_config *config = dev->config;
+
+	ret = i2c_write_dt(&config->bus, &reg, sizeof(reg));
+	if (ret < 0) {
+		return ret;
+	}
+	ret = i2c_read_dt(&config->bus, &reg, sizeof(reg));
+	if (ret < 0) {
+		return ret;
+	}
+
+	reg = enable ? (reg | OV76770_MVFP_VFLIP) : (reg & ~OV76770_MVFP_VFLIP);
+	return i2c_reg_write_byte_dt(&config->bus, OV7670_MVFP, reg);
+}
+
+static int ov7670_set_ctrl_hmirror(const struct device *dev, bool enable)
+{
+	int ret;
+	uint8_t reg = OV7670_MVFP;
+	const struct ov7670_config *config = dev->config;
+
+	ret = i2c_write_dt(&config->bus, &reg, sizeof(reg));
+	if (ret < 0) {
+		return ret;
+	}
+	ret = i2c_read_dt(&config->bus, &reg, sizeof(reg));
+	if (ret < 0) {
+		return ret;
+	}
+
+	reg = enable ? (reg | OV76770_MVFP_MIRROR) : (reg & ~OV76770_MVFP_MIRROR);
+	return i2c_reg_write_byte_dt(&config->bus, OV7670_MVFP, reg);
+}
```

Comment:
It might be convenient to have a "read/modify/write" function provided by Zephyr `video_common.c` some point in time... In the meantime, the way you did is the best IMHO.